changelog.txt

```diff
@@ -50,6 +50,7 @@ We recommend to go through the tutorial on signac-docs_ to learn how to best tak
 - Remove the Ascent environment (host decomissioned).
 - Fixes issue where callable cmd-directives were not evaluated.
 - Fixed University of Michigan Flux hostname pattern to ignore the Flux Hadoop cluster.
+- Fixes issue where using the buffered mode would lead to confusing error messages when condition-functions would raise an AttributeError exception.
```

Comment:
Do we want to reference github issue numbers in the change log?

Reply:
We have not done this so far, so I would say no for consistency sake.